src/strategies/tranche-staking/index.ts

```diff
@@ -0,0 +1,58 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'bonustrack';
```

Comment:
This should be you GitHub username
```suggestion
export const author = 'ayush-jibrel';
```

Reply:
Done